src/common/libscdl.c

```diff
@@ -64,10 +64,22 @@ int sc_dlclose(void *handle)
 	return FreeLibrary((HMODULE)handle);
 }
 #else
+#ifdef HAVE_DLMOPEN
+#define _GNU_SOURCE
+#endif
 #include <dlfcn.h>
 void *sc_dlopen(const char *filename)
 {
-	return dlopen(filename, RTLD_LAZY);
+	return dlmopen(LM_ID_NEWLM, filename, RTLD_LAZY | RTLD_LOCAL);
```

Comment:
this should stay dlopen, should not it?
```suggestion
	return dlopen(LM_ID_NEWLM, filename, RTLD_LAZY | RTLD_LOCAL);
```